### DISCUSSION TITLE
Hello, I tried to run like you said, getting main_leagues.pkl file error.

### DISCUSSION THREAD

**Discussion by speedspino21** [georgedouzas/sports-betting#38] [Q&A]
![Screenshot_1](https://github.com/georgedouzas/sports-betting/assets/61560191/2dc7587f-1628-46a1-b353-6c596b74e111)
Getting this error, kindly help me to resolve this problem.

**Comment by georgedouzas** [comment]
Hi speedspino21,

The directory `data` was missing. The new version 0.5.1 creates the directory and you will not get an error. Please try it and let me know!

**Comment by speedspino21** [comment]
Hello, where can I get version 0.5.1 , can you give me link? please? Thanks.

**Comment by georgedouzas** [comment]
Hi,

`pip install -U sports-betting` will install the latest version.

Alternative, if you want to clone repo (and the specific tag), use the following:

` git clone https://github.com/georgedouzas/sports-betting.git --branch 0.5.1 --single-branch`

**Comment by speedspino21** [comment]
I am getting pdm install problem now, sorry again..tried to fix, but not configured.
![Screenshot_1](https://github.com/georgedouzas/sports-betting/assets/61560191/794a9742-6e1e-40e7-b41f-cad83c96926c)

**Comment by georgedouzas** [comment]
This seems like a `pdm` installation issue and not `sports-betting`.

**Reply by speedspino21** [reply]
Do you have any youtube channel for manual guide how to use this and make dataloader file, and so on..

**Reply by georgedouzas** [reply]
A youtube video is a good idea. I will do it in a few days. In the meantime, have you checked the [documentation](https://georgedouzas.github.io/sports-betting)?

**Reply by speedspino21** [reply]
I had to input main_leaguge.py file because there was dataloader.py file, haha, yes, video will be good brother
I am looking forward it..

**Reply by speedspino21** [reply]
Brother, do you have any discord channel or live chatting tool? that's better , haha

**Reply by speedspino21** [reply]
I am still having so many question on this project
